vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -139,7 +139,8 @@ $global:map = @{
         "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
         "https://acs-mirror.azureedge.net/ccgakvplugin/v1.1.5/binaries/windows-gmsa-ccgakvplugin-v1.1.5.zip",
         "https://acs-mirror.azureedge.net/aks/windows/cse/aks-windows-cse-scripts-v0.0.12.zip",
```

Comment:
Can we remove aks-windows-cse-scripts-v0.0.12.zip?

Reply:
Removed. Thank you.